xc/device.ml

```diff
@@ -1404,6 +1404,8 @@ module Vusb = struct
     | _ ->()
 
   let vusb_plug ~xs ~domid ~id ~hostbus ~hostport ~version =
+    if not (is_upstream_qemu domid) then
+      raise (Internal_error (Printf.sprintf "Fail to plug, qemu-upstream must be used when plug usb."));
     debug "Vusb plugged: vusb device plugged ";
```

Comment:
This would be more helpful if the device would be named in the log message,

Reply:
updated.